osmaxx-py/osmaxx/excerptexport/static/excerptexport/libraries/Leaflet.Spin/leaflet.spin.js

```diff
@@ -0,0 +1,41 @@
+L.SpinMapMixin = {
+    spin: function (state, options) {
+        if (!!state) {
```

Comment:
Oops, I didn't notice that this was in `osmaxx/osmaxx-py/osmaxx/excerptexport/static/excerptexport/`**`libraries`** and thought this was a script written by us.